core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -335,6 +337,10 @@ angular.module('oppia').component('settingsTab', {
         AlertsService.clearWarnings();
 
         var moderatorEmailDraftUrl = '/moderatorhandler/email_draft';
+        var onUnpublishingcallback = function() {
+          UserExplorationPermissionsService.fetchPermissionsAsync();
+          ctrl.refreshPermissions();
```

Comment:
```suggestion
          UserExplorationPermissionsService.fetchPermissionsAsync().then(function(permissions) {
            ctrl.canUnpublish = permissions.canUnpublish;
            ctrl.canReleaseOwnership = permissions.canReleaseOwnership;
            // TODO(#8521): Remove the use of $rootScope.$apply()
            // once the controller is migrated to angular.
            $rootScope.$applyAsync();
          });
```

Can we do this considering refreshPermissions is also making the same call? (not sure whether refreshPermissions is also used anywhere else.]

Reply:
DONE